portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -169,4 +192,14 @@ private void deleteCommentWithId(String id) {
       datastore.delete(entity.getKey());
     }
   }
+
+  private class SortOrder {
+    String property;
+    SortDirection sortDirection;
+
+    public SortOrder(String property, SortDirection sortDirection) {
```

Comment:
Private class with a public constructor reads a little odd to me, maybe remove the modifier here?